src/miral/x11_support.cpp

```diff
@@ -36,7 +36,7 @@ void miral::X11Support::operator()(mir::Server& server) const
 {
     server.add_configuration_option(
         mo::x11_display_opt,
-        "DISPLAY socket to use for experimental X11 support (default: none).", mir::OptionType::integer);
+        "Enable DISPLAY socket to use for X11 support", mir::OptionType::null);
```

Comment:
The wording here is unclear. If the meaning is what I think it is, perhaps "Enable X11 support" would be better.